uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -150,7 +158,61 @@ <h4 class="modal-title">
                 </button>
               </div>
             </uib-tab>
-            <uib-tab heading="Files"></uib-tab>
+            <uib-tab heading="Files">
+              <div>
+                <ul class="chr_leave-request-files_list">
+                  <li>
```

Comment:
like the idea of spans so added, but not sure of empty class names for structure so left it